asv/benchmarks/cwt_benchmarks.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import pywt
+
+
+class CwtTimeSuiteBase(object):
+    """
+    Set-up for CWT timing.
+    """
+    params = ([32, 128, 512],
+              ['cmor', 'cgau4', 'fbsp', 'gaus4', 'mexh', 'morl', 'shan'],
+              [16, 64, 256])
+    param_names = ('n', 'wavelet', 'max_scale')
+
+    def setup(self, n, wavelet, max_scale):
+        try:
+            from pywt import cwt
+        except ImportError:
+            raise NotImplementedError("cwt not available")
```

Comment:
Is there an advantage to doing this within ``setup`` and raising an exception? SciPy does it like I described at https://github.com/PyWavelets/pywt/pull/158#r54647480. Here's an example for ``lgmres``, https://github.com/scipy/scipy/blob/master/benchmarks/benchmarks/sparse_linalg_solve.py#L15, works fine and seems simpler.